### DISCUSSION TITLE
Cannot see population simulations profile

### DISCUSSION THREAD

**Discussion by Cheruvh** [Open-Systems-Pharmacology/Forum#1628] [Questions & Problems]
Hi,

I got the individual PK mean profile simulation correct, however, when tried a population simulation the PK profile was blank as shown below:
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/103236850/17647951-9219-40ae-a173-e1b5c96f1efb)

I verified and found the ticks for the simulation concentrations, also the legends show them. Moreover, the x and y-axis limits are much lower even though the simulations were performed up to 270 h. Please suggest what needs to be adjusted here to visualize the graph. Thank you

**Comment by Yuri05** [comment]
Can you attach your project?

**Comment by NingJia012** [comment]
Hello, I encountered the same problem when conducting the rat group simulation. May I ask if you have solved this problem?
